packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -254,9 +256,16 @@ function overwriteStack(error: IFluidErrorBase | LoggingError, stack: string) {
  * False for any error we created and raised within the FF codebase, or wrapped in a well-known error type
```

Comment:
```suggestion
 * False for any error we created and raised within the FF codebase via LoggingError base class, or wrapped in a well-known error type
```

(probably exceeds max line len now)

Reply:
Sounds good. I have added the changes in commit #a1d49538.